documentation/openapi.yaml

```diff
@@ -71,22 +74,71 @@ paths:
                   username:
                     type: string
                     example: 's77777'
+                  primary_email:
+                    type: string
+                    example: 'foo@bar.de'
+                  alt_emails:
+                    type: array
+                    items:
+                      type: string
+                    example: ['hello@world.de','stein@beisser.de']
+                  admin:
+                    type: boolean
+                    example: true
                   portal_groups:
                     type: array
                     items:
                       type: integer
                     example: ['1', '2']
-                  service_account:
```

Comment:
so, you propose to use k8s_account instead of service_account? If we do this, I propose to implement that change in FE (recently in the welcome page", too, to avoid confusion. 
But I like the new field names. Vote for it.

Reply:
Agreed, service account is the internal K8S name most people don't know.